pages/posts/content.json

```diff
@@ -0,0 +1,13 @@
sion. You can set up or participate in ad-hoc groups while preserving privacy. And data is encrypted as it moves between users through an untrusted environment, whether device-to-device or over the Internet when available.</p><p>This protocol powers Berty Messenger which brings these features to mobile phone users. But the the need for reliable, privacy-based off-grid communication goes beyond a messing app. That’s why we’re introducing the open-source <a href='https://wesh.network'>Wesh Network Toolkit</a> so you can take advantage of the <a href='https://berty.tech/docs/protocol'>Wesh protocol</a> in your own application.</p> <h3>How will you use Wesh?</h3> For maximum flexibility, your application interfaces to Wesh based on <a href='https://grpc.io'>gRPC</a>, a well-supported framework which sends and receives Google Protobuf messages.<ul> <li>Even though the <a href='https://github.com/berty/weshnet'>core Wesh code</a> is written in Go, gRPC messages are language-independent. So, Wesh works with your existing application written in Go, Python, Java, C++ or <a href='https://grpc.io/docs/languages'>other supported languages</a>.</li> <li>gRPC works with the Buf schema registry where we publish the <a href='https://buf.build/berty/weshnet'>Wesh API</a>. For example, this documents the request and reply to <a href='https://buf.build/berty/weshnet/docs/main:weshnet.protocol.v1#weshnet.protocol.v1.ProtocolService.GroupInfo'>retrieve information about a group</a>.</li> <li>gRPC allows different calling interfaces. The messages can be sent as if Wesh is a library linked directly to your application. Or if your application environment requires it, then API messages can be sent over TCP to a Wesh toolkit running on a local or remote server.</li></ul> <h3>The “Wesh” name</h3>You may be wondering what “Wesh” means. It’s a variant of “mesh” to highlight its asynchronous mesh communication which can use any device to securely store a message for a disconnected group member and deliver it when they connect again. (”Wesh” is also slang for “hi” in the French hip hop scene.) <h3>What’s next?</h3>This was a short and sweet introduction to Wesh. In future posts, we’ll dive into how Wesh is organized by users and groups, how to get started with the code, some example applications and more. Stay tuned! <h3>Links</h3><ul><li><a href='https://wesh.network'>https://wesh.network</a>  The main Wesh Network Toolkit web page</li><li><a href='https://github.com/berty/weshnet'>https://github.com/berty/weshnet</a>  The Wesh GitHub repo for code and issues</li><li><a href='https://berty.tech/docs/protocol'>https://berty.tech/docs/protocol</a>  The Wesh protocol white paper</li><li><a href='https://buf.build/berty/weshnet'>https://buf.build/berty/weshnet</a>  The Wesh API at the Buf schema registry</li><li><a href='https://berty.tech/about'>https://berty.tech/about</a> About Berty Technologies, the NGO working with the open-source community to make all this real</li></ul>",
```

Comment:
"But the the need for reliable" should be "But the need for reliable". (One "the".) Also, "beyond a messing app" should be "beyond a messaging app". (There errors were in the original Notion page. Sorry! :-)